app-text/zathura/zathura-9999.ebuild

```diff
@@ -1,44 +1,41 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 inherit meson virtualx xdg
 
 DESCRIPTION="A highly customizable and functional document viewer"
 HOMEPAGE="https://pwmt.org/projects/zathura/"
 
-if [[ ${PV} == *9999 ]]; then
```

Comment:
I personally prefer keeping "live templates" if we're keeping the live ebuild because it makes it far easier to keep the two in sync (live ebuilds have a tendency to rot otherwise).